pkgs/development/python-modules/openapi-python-client/default.nix

```diff
@@ -0,0 +1,72 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  autoflake,
+  poetry-core,
+  pyyaml,
+  typer,
+  colorama,
+  httpx,
+  python-dateutil,
+  jinja2,
+  black,
+  pydantic,
+  isort,
+  shellingham,
+  pytest-mock,
+  pytestCheckHook,
+  openapi-python-client-integration-tests,
+}:
+
+buildPythonPackage rec {
+  pname = "openapi-python-client";
+  version = "0.14.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "openapi-generators";
+    repo = "openapi-python-client";
+    rev = "v${version}";
+    hash = "sha256-Etw0Iz5aAdu/K+WSmue9ux8qi0f/YKgZgVhqCbGVxE0=";
+  };
+
+  nativeBuildInputs = [
+    colorama
+    httpx
+    jinja2
+    poetry-core
+    python-dateutil
+    pyyaml
+    shellingham
+    typer
+    openapi-python-client-integration-tests
+  ];
+
+  propagatedBuildInputs = [
+    autoflake
+    black
+    isort
+    pydantic
+  ];
```

Comment:
This seems to be not quite right. I think you maybe mixed up runtime and build time dependencies or the package is not complete standard. Also we don't want to have any linters in here unless the package imports code from them and not just uses their lints.

Reply:
I don't do python development, so I don't know which package are related to linters.
I'm just trying to replicate what I see elsewhere.
Thanks for your patience with this.

Reply:
If you don't know the packages, you can give them a quick google and it should be rather obvious that they are linting.